File: beecrowd/python/1018.py
N = int(input())
print (N)
cem = N // 100
N = N % 100
cinq = N // 50
N = N % 50
vin = N // 20
N = N % 20
dez = N // 10
N = N % 10
cin = N // 5
N = N % 5
dois = N//2
N = N % 2
print("%i nota(s) de R$ 100,00" % cem)
print("%i nota(s) de R$ 50,00" % cinq)
print("%i nota(s) de R$ 20,00" % vin)
print("%i nota(s) de R$ 10,00" % dez)
print("%i nota(s) de R$ 5,00" % cin)
print("%i nota(s) de R$ 2,00" % dois)
print("%i nota(s) de R$ 1,00" % N)

File: beecrowd/java/1018.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int x, cem,cinq,vin,dez,cin,dois;
        x = sc.nextInt();
        System.out.printf("%d\n", x);
        cem = x / 100;
        x = x % 100;
        cinq = x / 50;
        x = x % 50;
        vin = x / 20;
        x = x % 20;
        dez = x / 10;
        x = x % 10;
        cin = x / 5;
        x = x % 5;
        dois = x/2;
        x = x % 2;
        System.out.printf("%d nota(s) de R$ 100,00\n", cem);
        System.out.printf("%d nota(s) de R$ 50,00\n", cinq);
        System.out.printf("%d nota(s) de R$ 20,00\n", vin);
        System.out.printf("%d nota(s) de R$ 10,00\n", dez);
        System.out.printf("%d nota(s) de R$ 5,00\n", cin);
        System.out.printf("%d nota(s) de R$ 2,00\n", dois);
        System.out.printf("%d nota(s) de R$ 1,00\n", x);
    }
}